SummerInternship.md

```diff
@@ -1,15 +1,43 @@
-<This pie chart tells us that out of all the MADE shots during the 2018-19 NBA season, 47% of the makes were layups, 42% of the makes were jumpers, 6% of the makes were floaters(in between a jumper and a layup), 5% of the makes were post shots, and 0% of the makes were heaves. This analysis is expected because layups are attempted closest to the basket and thus have the highest percentage of going in. On the other hand, no heaves were made at all because they are desperation shots attempted extremely far from the basket(usually when the clock is winding down), thus making them nearly impossible to make.>             
-Test from Sarah
+  
+This pie chart tells us that out of all the MADE field goals during the 2018-19 NBA season, 47% of the makes were layups, 42% of the makes were jumpers, 6% of the makes were floaters(in between a jumper and a layup), 5% of the makes were post shots, and 0% of the makes were heaves. This analysis is expected because layups are attempted closest to the basket and thus have the highest percentage of going in. On the other hand, no heaves were made at all because they are desperation shots attempted extremely far from the basket(usually when the clock is winding down), thus making them nearly impossible to make. 
+
+
+
+#### Shot-Type Based on Missed Shots
+<img width="540" alt="Screen Shot 2019-08-13 at 1 34 14 PM" src="https://user-images.githubusercontent.com/51301795/63013385-fdfe5a00-be40-11e9-8d46-d5ab73c1a6d6.png">
+
+This pie chart tells us that out of all the MISSED field goals during the 2018-19 NBA season, 54% of the misses were jumpers while 33% of the misses were layups. This makes sense because jumpers (which include threes) are taken farther from the basket than layups but are attempted extremely often, thus making it the most commonly missed shot. At first it was very surprising to see that 33% of missed shots were layups since layups are presumably easy, close range shots (as opposed to heaves, for example, which you would think are a harder, more commonly missed shots). I predict that this is because layups are attempted in traffic most of the time, meaning that these shots are heavily contested by bigger defenders under the basket and thus have a great chance of being blocked or tipped (which equals a miss). Additionally, I assume that players draw the most fouls off of layups since since driving to the basket requires an immense amount of physicality; in other words, players miss layups more than you would think because they are under duress. I also speculate that players don't always attempt shots to score but rather to draw a foul, thus sending them to the free throw line for two (or three) shots - after all, you have a better chance of making two uncontested, unhurried foul shots compared to a contested and likely off-balance jumper. This is yet another reason why jumpers and layups are so commonly missed.
+
```

Comment:
```suggestion
##### Future Graphs
It would be really interesting to know:
Based on one type of jump, what is the likelihood of close made vs miss and far made vs miss
```